app/uk/gov/hmrc/gform/views/form/form.scala.html

```diff
@@ -85,15 +84,15 @@
     @pageLevelError.render
 
     @if(shouldDisplayHeading) {
-      @hmrcPageHeading(PageHeading(text = heading, section = page.progressIndicator))
+      @pageHeading(heading, page.progressIndicator)
 
       @if(page.sectionDescription.exists(_.trim.nonEmpty)){
         <p class="govuk-body">@{page.sectionDescription.get}</p>
       }
     } else {
       @* This is here to support caption in case of isPageHading case *@
       @page.progressIndicator.fold(HtmlFormat.empty){ caption =>
-        <span class="govuk-caption-xl">@caption</span>
+        <p class="govuk-!-margin-bottom-0 govuk-caption-m hmrc-caption-xl"><span class="govuk-visually-hidden">This section is </span>@caption</p>
```

Comment:
this is another instance of the `messages("span.key")`.

Also `span.key` seems like a bad name,  cannot we use something more descriptive for instance `this.section.is` :)